src/pages/orders/OrderRow.tsx

```diff
@@ -1,53 +1,28 @@
 import React from 'react';
-import gql from 'graphql-tag';
-import { OrderRow_order } from './__generated__/OrderRow_order';
+import { OrderRow_orderFragment } from './__generated__/OrderRow_order';
```

Comment:
**3) Will be nice `typescript-react-apollo` allows to turn off `Fragment` suffix** for generated types eg `OrderRow_orderFragment` -> `OrderRow_order`. [relay-compiler enforces fragments to be named as `<FileName>_<propName>`](https://relay.dev/docs/en/fragment-container#arguments). And it's is very useful and cool!

Reply:
Hmm we wanted to change that, but it will cause a breaking change for everyone who's using it this way. Maybe in the next major release :)

Reply:
Maybe add a new option `omitOperationSuffix` which will be `false` by default. And it helps to keep compatibility.